src/App.css

```diff
@@ -1,19 +1,33 @@
-main {
+.app-body {
   display: grid;
-  grid-template-columns: 600px auto auto;
-  grid-template-rows: 150px 100px auto 300px;
+  grid-template-columns: 200px 700px auto;
+  grid-template-rows: 100px 100px auto 300px;
 }
 .header {
   grid-column: 1 / 4;
   grid-row: 1 / 2;
   border-color: black;
+  background-color:aliceblue;
   padding: 2rem;
   margin: 1rem;
 }
 .article-feed {
   grid-column: 2 / 3;
   grid-row: 2 / 5;
   border-color: black;
+  background-color:aliceblue;
+  height: 500px;
+  overflow: scroll;
+  padding: 2rem;
+  margin: 1rem;
+}
+.comments-container {
+  grid-column: 3 / 4;
+  grid-row: 2 / 4;
```

Comment:
The comments section is really small. This should be given more screen space to read them properly. Make it take the rest of the page at least.